api/namespace/applications/routes.py

```diff
@@ -23,6 +22,9 @@ class SearchApplications(Resource):
     query_params_parser.add_argument(
         "fund_id", type=str, help="Application fund_id"
     )
+    query_params_parser.add_argument(
+        "user_id", type=str, help="Application user_id"
```

Comment:
I think the term 'account_id' is being used in other places (i.e. the id generated in the account store)?

Reply:
@harryyo I have forced pushed to make this change (and also various linting things now I have the pre-commit hooks setup)